src/Components/Endpoints/src/RazorComponentEndpointInvoker.cs

```diff
@@ -133,7 +141,12 @@ private async Task RenderComponentCore(HttpContext context)
 
     private async Task<RequestValidationState> ValidateRequestAsync(HttpContext context, IAntiforgery? antiforgery)
     {
-        var isPost = HttpMethods.IsPost(context.Request.Method);
+        var isPost = HttpMethods.IsPost(context.Request.Method) &&
```

Comment:
```suggestion
        var processPost = HttpMethods.IsPost(context.Request.Method) &&
```

since it still can be a post even if it's an error